apps/etterlatte-saksbehandling-ui/client/src/components/person/personopplysninger/Personopplysninger.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { ReactNode, useEffect, useState } from 'react'
+import { Container, SpaceChildren } from '~shared/styled'
+import { LenkeTilAndreSystemer } from '~components/person/personopplysninger/LenkeTilAndreSystemer'
+import { Bostedsadresser } from '~components/person/personopplysninger/Bostedsadresser'
+import { isSuccess, mapResult, Result } from '~shared/api/apiUtils'
+import { SakMedBehandlinger } from '~components/person/typer'
+import { useApiCall } from '~shared/hooks/useApiCall'
+import { hentPersonopplysningerForBehandling } from '~shared/api/grunnlag'
+import Spinner from '~shared/Spinner'
+import { ApiErrorAlert } from '~ErrorBoundary'
+import { Statsborgerskap } from '~components/person/personopplysninger/Statsborgerskap'
+import { Alert, Heading } from '@navikt/ds-react'
+import { SakType } from '~shared/types/sak'
+import { Foreldre } from '~components/person/personopplysninger/Foreldre'
+import { AvdoedesBarn } from '~components/person/personopplysninger/AvdoedesBarn'
+import { Sivilstatus } from '~components/person/personopplysninger/Sivilstatus'
+import { Innflytting } from '~components/person/personopplysninger/Innflytting'
+import { hentAlleLand, ILand } from '~shared/api/trygdetid'
+import { Utflytting } from '~components/person/personopplysninger/Utflytting'
+import { Vergemaal } from '~components/person/personopplysninger/Vergemaal'
+
+export const Personopplysninger = ({
+  sakStatus,
+  fnr,
+}: {
+  sakStatus: Result<SakMedBehandlinger>
+  fnr: string
+}): ReactNode => {
+  const [landListe, setLandListe] = useState<ILand[]>([])
+  const [harBehandling, setHarBehandling] = useState<boolean>(false)
+
+  const [personopplysningerResult, hentPersonopplysninger] = useApiCall(hentPersonopplysningerForBehandling)
+  const [, hentLandListe] = useApiCall(hentAlleLand)
+
+  const erSaktype = (sakStatus: Result<SakMedBehandlinger>, sakType: SakType) => {
+    return isSuccess(sakStatus) && sakStatus.data.sak.sakType === sakType
+  }
+
+  useEffect(() => {
+    if (isSuccess(sakStatus)) {
+      if (sakStatus.data.behandlinger.length > 0) {
+        setHarBehandling(true)
+        hentPersonopplysninger({
+          behandlingId: sakStatus.data.behandlinger[0].id,
+          sakType: sakStatus.data.behandlinger[0].sakType,
+        })
+      }
+    }
+  }, [fnr, sakStatus])
+
+  useEffect(() => {
+    hentLandListe(null, setLandListe)
+  }, [])
+
+  return (
+    <Container>
+      <SpaceChildren>
+        {!!sakStatus && (
```

Comment:
`sakStatus` er så og si alltid satt, så denne sjekken har lite for seg. Bruk heller tilhørende `apiUtils`-hjelpefunksjoner for å sjekke korrekt tilstand